en/lessons/advanced/otp-distribution.md

```diff
@@ -0,0 +1,430 @@
+---
+version: 1.0.0
+title: OTP Distribution
+---
+## Introduction to Distribution
+We can run our Elixir apps on a set of different nodes distributed across a single host or across multiple hosts. Elixir allows us to communicate across these nodes via a few different mechanisms which we will outline in this lesson.
+
+{% include toc.html %}
+
+## Communication Between Nodes
+
+Elixir runs on the Erlang VM, which means it has access to to Erlang's powerful [distribution functionality.](http://erlang.org/doc/reference_manual/distributed.html)
+
+> A distributed Erlang system consists of a number of Erlang runtime systems communicating with each other. Each such runtime system is called a node.
+
+A node is any Erlang runtime system that has been given a name. We can start a node by opening up `iex` session and naming it:
+
+```bash
+iex --sname alex@localhost
+iex(alex@localhost)1>
```

Comment:
We don't show `iex` line numbers (I disabled them via config for myself), it's mentioned in the styleguide.